aws-api/src/main/java/com/amplifyframework/api/aws/AppSyncGraphQLOperation.java

```diff
@@ -86,8 +86,8 @@ private AppSyncGraphQLOperation(
 
     @Override
     public void start() {
-        // No-op if start() is called post-execution
-        if (ongoingCall != null && ongoingCall.isExecuted()) {
+        // No-op if start() is called post-execution or canceled
+        if (ongoingCall != null && ((ongoingCall.isExecuted()) || ongoingCall.isCanceled())) {
```

Comment:
```suggestion
        if (ongoingCall != null && (ongoingCall.isExecuted() || ongoingCall.isCanceled())) {
```
nitpicking, but there was an extra set of parentheses there :)

Reply:
Thanks, will fix.